### DISCUSSION TITLE
[FEATURE] Why is the tabler-vue project archived

### DISCUSSION THREAD

**Discussion by taoweidong** [tabler/tabler#1483] [Q&A] [answered]
Does this framework support the vue project? Why is the tabler-vue project archived?

**Comment by rjd22** [comment]
I've been using tabler with Vue3 and you don't really need to do much to make it work. I've written my own components for the parts that I use and use the styles from tabler.

I think it's archived because supporting all different JS frameworks is just a lot of work and requires a lot of knowledge on all of them.

**Comment by codecalm** [comment, marked as answer]
We're starting to work at official Vue package for Tabler :)

**Reply by cuiweiqiang** [reply]
Hi, Is there any update?

**Reply by stephenr85** [reply]
Also looking for this!

**Comment by aruiz122ahnnf** [comment]
ok i did all the stuff that you wanted me to do
